README.md

```diff
@@ -88,23 +38,357 @@ Following is a summary of the values you must provide in the templated functions
 -   `NEG_NAME`: Name of the PSC network-endpoint-group for apigee
     service-attachment. ***[Required only when using PSC NEG]***
 
-### 2. Provision the new, temporary instance
+# Apigee-X Instance Recreation with downtime and data loss.
```

Comment:
Add a brief summary on the options available before deep diving into each of them.

Reply:
After looking at each of the option, I think lets move each option to different readme file. In the top level readme (this file), lets discuss the options briefly and the allow customer to chose an option and proceed. Chosing an option should take them to corresponding readme. 

Having everything in this file will cause confusion and provides a look of complex procedure.